config.yaml

```diff
@@ -1,8 +1,4 @@
 options:
-    advertised-port:
```

Comment:
https://github.com/relaxdiego/charm-k8s-prometheus/blob/master/src/interface_http.py#L36 will break then, as there will be no `advertised-port` config option.

Reply:
Ah thanks for pointing that out. I've fixed it with a follow-up commit below.